src/components/header/Header.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import '../body/RoutesApp/RoutesApp.css';
+
+function Header() {
+    const navStyle = {
+        color: 'white'
+    };
+    return (
+        <nav className="navbar navbar-expand-lg navbar-light bg-light">
+            <a className="navbar-brand" href="#">Aruba HTS</a>
+            <button className="navbar-toggler" type="button" data-toggle="collapse"
+                    data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false"
+                    aria-label="Toggle navigation">
+                <span className="navbar-toggler-icon"></span>
+            </button>
+
+            <div className="collapse navbar-collapse" id="navbarSupportedContent">
```

Comment:
Is this class 'collapse navbar-collapse' makes a the drop-down in the navBar. Could you please explain and make me understand?

Reply:
no, its for collapsing in mobile view